l1-contracts/contracts/state-transition/chain-deps/DiamondInit.sol

```diff
@@ -66,6 +66,13 @@ contract DiamondInit is ZkSyncHyperchainBase, IDiamondInit {
         s.feeParams = _initializeData.feeParams;
         s.blobVersionedHashRetriever = _initializeData.blobVersionedHashRetriever;
 
+        if (_initializeData.batchCommitDeadline == 0) {
+            s.batchCommitDeadline = 3 days;
```

Comment:
let's always set to 3 days, note that we enforce that all chains use the same `_initializeData`